src/deps.ts

```diff
@@ -41,9 +40,35 @@ export interface KerberosClient {
   unwrap: (challenge: string, callback?: Callback<string>) => Promise<string> | void;
 }
 
-export let Snappy:
-  | typeof import('snappy')
-  | { kModuleError: MongoMissingDependencyError } = makeErrorModule(
+type SnappyLib = {
```

Comment:
Defining our own types here lets us pass in that callback that may or may not exist.